### ISSUE TITLE
Inconsistencies in <strong> and <em> conversions

### ISSUE THREAD

**Issue by dsoaress** [crosstype/node-html-markdown#9]
Hi, I noticed two inconsistencies in the conversion when the HTML source has `<strong>` and `<em>`. The generated markdown comes without the before and after spaces.

Examples: 

```html
<!-- html source -->
<p>Hello <strong>world</strong>!</p>
```
```markdown
<!-- markdown -->
Hello**world**!
```

```html
<!-- html source -->
<p>Lorem <em>ipsum</em> dolor sit.</p>
```
```markdown
<!-- markdown -->
Lorem*ipsum*dolor sit.

<!-- 
    I put the emphasis as an asterisk here in the 
    example just to enable the syntax highlighting 
    here from GitHub 
-->
```

I'm using it without any options enabled.

```js
NodeHtmlMarkdown.translate(rawBody)
```

**Comment by nonara** [comment]
That definitely shouldn't happen! Thanks for reporting it. Looking into it now/

**Comment by nonara** [comment]
Looks like the issue is upstream, in the parser. I've filed an issue https://github.com/taoqf/node-html-parser/issues/115

If it can't be addressed quickly, I'll see if I can submit a PR. Will post updates here as I get them

**Comment by nonara** [comment]
I have a PR in the pipeline. Hopefully we'll have this sorted soon.

**Comment by nonara** [comment]
Sorry for the delay on this. I had the fix sitting there and thought I'd pushed it through. Fixed in v0.1.7
